libraries/networking/src/FileResourceRequest.cpp

```diff
@@ -18,17 +18,15 @@ void FileResourceRequest::doSend() {
     
     QFile file(filename);
     
-    _state = Finished;
-    if (file.exists()) {
-        if (file.open(QFile::ReadOnly)) {
-            _data = file.readAll();
-            _result = ResourceRequest::Success;
-        } else {
-            _result = ResourceRequest::AccessDenied;
-        }
+    if (!file.exists()) {
+        _result = NotFound;
+    } else if (file.open(QFile::ReadOnly)) {
+        _data = file.readAll();
+        _result = Success;
     } else {
-        _result = ResourceRequest::NotFound;
+        _result = AccessDenied;
     }
```

Comment:
I don't feel like I can authoritatively say whether it is or isn't, but both contain 2 branches so I'm not sure where it would be less performant wrt branching. I guess it contains an extra JMP? But looking at that is seriously premature optimization territory.